app/views/landings/index.html.erb

```diff
@@ -13,9 +13,9 @@
     <div class="ui segment">
       <div class="ui header">
         <%= je.position %> at <%= je.company %>
-      </div>
-      <div class="ui label">
-        Location: <%= je.city %>, <%= je.state %>
+        <% if @current_user %>
+          <div class="ui right floated button"><%= link_to "Edit", edit_job_experience_path(je) %></div>
```

Comment:
Same here. Text is blue because it's a link, while it's gray for the Update button on the edit page.